From bfecf657e27b9e0705ca584382a885eae47e733e Mon Sep 17 00:00:00 2001
From: Yotam Ben Dosa <ybendosa@habana.ai>
Date: Thu, 26 Dec 2019 15:49:02 +0200
Subject: [PATCH] [SW-0] make: hls: accommodate mainline openbmc

Change OpenBMC definitions and build instruction in order to support
mainline version.

Change-Id: I672f9e2e9db8f73bc4b8a8db81921af3c79d97e9
Signed-off-by: Yotam Ben Dosa <ybendosa@habana.ai>
---

diff --git a/package/openbmc/openbmc.mk b/package/openbmc/openbmc.mk
index 7f78a30..f738a27 100644
--- a/package/openbmc/openbmc.mk
+++ b/package/openbmc/openbmc.mk
@@ -23,9 +23,9 @@
 define OPENBMC_BUILD_CMDS
 	(cd $(@D); 							\
 		$(SET_VERSION_INFO);					\
-		$(OPENBMC_ENV)						\
+		$(OPENBMC_ENV);						\
 		$(OPENBMC_BITBAKE_ENV)					\
-		bitbake hls-image;					\
+		bitbake obmc-phosphor-image;				\
 	)
 endef
 
diff --git a/scripts/make/hls/Makefile b/scripts/make/hls/Makefile
index e968a04..64afae3 100644
--- a/scripts/make/hls/Makefile
+++ b/scripts/make/hls/Makefile
@@ -1,5 +1,5 @@
-export OPENBMC_WORK_DIR = build-habana
-export OPENBMC_ENV_FLAGS = meta-habana/meta-hls $(OPENBMC_WORK_DIR)
+export OPENBMC_WORK_DIR = build
+export OPENBMC_ENV_FLAGS = meta-habana/meta-hls/conf
 export OPENBMC_IMAGE_NAME = flash-hls
 export OPENBMC_IMAGE_DIR = $(OPENBMC_WORK_DIR)/tmp/deploy/images/hls
 export OPENBMC_DEPLOY_DIR = $(OUTPUT_DIR)/build/openbmc-custom
@@ -14,8 +14,8 @@
 export OPENBMC_BITBAKE_ENV
 
 define OPENBMC_ENV
-	mkdir -p $(OPENBMC_WORK_DIR);					\
-	source ./openbmc-init-build-env $(OPENBMC_ENV_FLAGS);
+	mkdir -p $(OPENBMC_WORK_DIR);				\
+	TEMPLATECONF=$(OPENBMC_ENV_FLAGS) source ./openbmc-env
 endef
 export OPENBMC_ENV
 
